### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by samdoran**
## Description
- Remove some inconistencies around PostgreSQL installation
- Favor Trino workflow
- Fix some broken links
- Correct file name
- Install and configure dependencies for `psycopg2`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -79,21 +73,41 @@ do the following:
...
 #### Starting Koku using Docker Compose
 
 > **Note**
 >
 > In order for the `koku_base` image to build correctly, buildkit must be enabled by setting `DOCKER_BUILDKIT=1`. This is set in the `.env` file, but if you are having issues building the `koku_base` image, make sure buildkit is enabled.
 
-1.  Start the docker containers:
+1.  Start the containers:
+
+        make docker-up-min-trino
```
Comment (maskarb) [inline README.md:86]: The `docker-up-min-trino` and `docker-up-min-trino-no-build` are now redundant and can be removed. The `docker-up-min` and `docker-up-min-no-build` include spinning up hive-metastore and trino.

File: README.md [hunk 2]
```diff
@@ -79,21 +73,41 @@ do the following:
...
+2.  Display log output from the containers. It is recommended that logs be kept in a second terminal:
+
+        make docker-logs
+
+With all containers running any source added will be processed by saving CSV files in MinIO and storing Parquet files in MinIO. The source's data will be summarized via Trino. Summarized data will land in the appropriate daily_summary table for the source type for consumption by the API.
+
+To add test sources and data:
+
+    make create-test-customer
+    make load-test-customer-data (optional)start={start_date} (optional)end={end_date}
```
Comment (maskarb) [inline README.md:97]: Could also specify a test_source, e.g. `test_source=AWS`

**Review by maskarb — COMMENTED** [review]
a couple comments.

**Review by kholdaway — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (samdoran) [reply README.md:86]: I was wondering about this. Thanks for clarifying.

File: README.md (hunk 2 above)
Comment (samdoran) [reply README.md:97]: Great idea. I use that quite a bit.

File: README.md [hunk 3]
```diff
@@ -21,15 +21,10 @@ This project is developed using Python 3.9. Make sure you have at least this ver
 ### Prerequisites
 
 -   Docker
--   PostgreSQL
 
-#### For Mac OSX
+#### For macOS
 
--   [Install Docker for Mac](https://docs.docker.com/v17.12/docker-for-mac/install/)
```
[the remark this answers is not in the document]
Comment (samdoran) [reply README.md:28]: I'll combine this into the Prerequisites section.

**Review by lcouzens — DISMISSED** [review]

**Review by kholdaway — APPROVED** [review]
Looks good

File: README.md [hunk 4]
```diff
@@ -12,24 +12,16 @@ Koku's goal is to provide an open source solution for cost management of cloud a
...
--   Docker
--   PostgreSQL
-
-#### For Mac OSX
-
--   [Install Docker for Mac](https://docs.docker.com/v17.12/docker-for-mac/install/)
--   [Install brew](https://brew.sh/)
--   Install PostgreSQL:
-
-        brew install postgresql
```
[the remark this answers is not in the document]
Comment (kholdaway) [reply README.md:32]: Perfect

**Pull request merged**